docs: update article to describe new methods in ErrorEvent (#2948)
vaadin/docs: articles/advanced/custom-error-handler.asciidoc

To apply the custom error handler to all user sessions, you can use a [classname]`SessionInitListener`, which receives an event each time a new [classname]`VaadinSession` is initialized. See the <<session-and-ui-init-listener#,Session and UI Listeners>> documentation page to learn how to create a [classname]`SessionInitListener`.

[role="since:com.vaadin:vaadin@V24.3"]

[classname]`ErrorEvent` has [methodname]`getComponent()` and [methodname]`getElement()` methods to get optionally a [classname]`Component` or [classname]`Element` that error was thrown for.

> For example, following button click listener would end up [methodname]`getComponent()` in [classname]`ErrorEvent` to return the button that was clicked:

Reviewer: ```suggestion
For example, following button click listener would have [classname]`ErrorEvent` [methodname]`getComponent()` return the clicked button:
```
